pkg/syncer/types/commontypes.go

```diff
@@ -57,32 +57,38 @@ func InitConfigInfo(ctx context.Context) (*ConfigInfo, error) {
 
 // GetVirtualCenterInstance returns the vcenter object singleton.
 // It is thread safe.
```

Comment:
Can you add some documentation about the usage of `reloadConfig` function param?

Reply:
Added.